index.html

```diff
@@ -0,0 +1,612 @@
+			</div>
+			<div class="right-nav-bar">
+				<button class="blue-btn">SUBSCRIBE</button> <button class="blue-btn">LOGIN</button>
+			</div>
+    </nav>
+    <hr class="nav-line">
+	</header>
+	<main>
+		<h2 class="space-ripples">Space Ripples Reveal<br>
+		Big Bang's Smoking Gun</h2>
+		<figure>
+			<img alt="big picture" class="big-picture" src="https://static01.nyt.com/images/2014/03/18/science/space/18cosmos/18cosmos-superJumbo.jpg?quality=90&auto=webp">
+			<figcaption class="alan-guth">
+				Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the<br>
+				universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago. <small>Rick<br>
+				Friedman for The New York Times</small>
+			</figcaption>
+		</figure>
+		<p class="by">By <a href="#"><span class="by-span">Dennis Overbye</span></a></p>
+		<div class="social-bar-container">
+			<div class="date">
+				<time datetime="2008-02-14 20:00">March 17, 2014</time>
+			</div>
+			<div class="left-social-bar">
+				<ul class="social-icons">
+					<li class="icons">
+						<a href="#"><i class="fab fa-facebook-f"></i></a>
+					</li>
+					<li class="icons">
+						<a href="#"><i class="fab fa-twitter"></i></a>
+					</li>
+					<li class="icons">
+						<a href="#"><i class="fas fa-envelope"></i></a>
+					</li>
+					<li class="icons">
+						<a href="#"><i class="fas fa-share"></i></a>
+					</li>
+					<li class="icons">
+						<a href="#"><svg class="css-ubw31m" height="25px" viewbox="0 0 16 25" width="16px">
```

Comment:
- [ ]  put height and width selectors in a class or Id in your CSS file as no inline styling is permitted in the HTML